aspnetcore/fundamentals/configuration/index.md

```diff
@@ -1,441 +1,365 @@
+The following commands test the custom prefix:
 
-```csharp
-.ConfigureAppConfiguration((hostingContext, config) =>
-{
-    config.Sources.Clear();
-    // Add providers here
-})
+```cmd
+set MyCustomPrefix_MyKey="My key with MyCustomPrefix_ Environment"
+set MyCustomPrefix_Position__Title=Editor_with_customPrefix
+set MyCustomPrefix_Position__Name=Environment_Rick_cp
+dotnet run
 ```
 
-### Consume configuration during app startup
+The [default configuration](#default) loads environment variables and command line arguments prefixed with `DOTNET_` and `ASPNETCORE_`. The `DOTNET_` and `ASPNETCORE_` are used by ASP.NET Core for [host and app configuration](xref:fundamentals/host/generic-host#host-configuration), not for user configuration. For more information on host and app configuration, see [.NET Generic Host](xref:fundamentals/host/generic-host).
 
-Configuration supplied to the app in `ConfigureAppConfiguration` is available during the app's startup, including `Startup.ConfigureServices`. For more information, see the [Access configuration during startup](#access-configuration-during-startup) section.
+On [Azure App Service](https://azure.microsoft.com/services/app-service/), select **New application setting** on the **Settings > Configuration** page. Azure App Service application settings are:
 
-## Command-line Configuration Provider
+* Encrypted at rest and transmitted over an encrypted channel.
+* Exposed as environment variables.
 
-The <xref:Microsoft.Extensions.Configuration.CommandLine.CommandLineConfigurationProvider> loads configuration from command-line argument key-value pairs at runtime.
+For more information, see [Azure Apps: Override app configuration using the Azure Portal](xref:host-and-deploy/azure-apps/index#override-app-configuration-using-the-azure-portal).
 
-To activate command-line configuration, the <xref:Microsoft.Extensions.Configuration.CommandLineConfigurationExtensions.AddCommandLine*> extension method is called on an instance of <xref:Microsoft.Extensions.Configuration.ConfigurationBuilder>.
+See [Connection string prefixes](#constr) for information on Azure database connection strings.
 
-`AddCommandLine` is automatically called when `CreateDefaultBuilder(string [])` is called. For more information, see the [Default configuration](#default-configuration) section.
+<a name="clcp"></a>
 
-`CreateDefaultBuilder` also loads:
+## Command-line
 
-* Optional configuration from *appsettings.json* and *appsettings.{Environment}.json* files.
+By [default](#default), the <xref:Microsoft.Extensions.Configuration.CommandLine.CommandLineConfigurationProvider> loads configuration from command-line argument key-value pairs after the following configuration sources:
+
+* *appsettings.json* and *appsettings*.`Environment`.*json* files.
 * [User secrets (Secret Manager)](xref:security/app-secrets) in the Development environment.
```

Comment:
I've seen this refered to as "secret manager", "app secrets", and "user secrets" in this topic. It would be good to make that a bit more consistent.